batch-jberet/src/main/java/org/wildfly/extension/batch/jberet/_private/Capabilities.java

```diff
@@ -39,10 +39,17 @@
      */
     public static final String DATA_SOURCE_CAPABILITY = "org.wildfly.data-source";
 
+    /**
+     * The capability name for the transaction manager.
+     */
+    public static final String TRANSACTION_MANAGER_CAPABILITY = "org.wildfly.transactions.transaction-manager";
+
     /**
      * A capability for the current batch configuration.
      */
     public static final RuntimeCapability<Void> BATCH_CONFIGURATION_CAPABILITY = RuntimeCapability.Builder.of("org.wildfly.batch.configuration", false, BatchConfiguration.class)
+            // BatchConfiguration itself doesn't require a TM, but any effective use of it does (BatchEnvironment)
+            .addRequirements(TRANSACTION_MANAGER_CAPABILITY)
```

Comment:
As I said in the JIRA, I don't think it's appropriate to have/use a TM capability.  A TM is _always_ available even when there is no transaction manager in the application server.  It'd be better to have a `org.wildfly.transaction.client.spi.LocalTransactionProvider` typed capability, or more particularly, a capability that indicates that there is a global transaction provider installed.  Things which use a TM but don't care whether or not the TM actually works do not need to rely on the capability at all (other than start order considerations).